recipes/r-bart/meta.yaml

```
@@ -0,0 +1,92 @@
+{% set version = '2.9.6' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-bart
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/BART_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/BART/BART_{{ version }}.tar.gz
+  sha256: 6e23eac476e894c0e6540eaa80e8436ea04261ce6aaac8d4f6f8f7ceeb082e0f
+
+build:
+  merge_build_host: True  # [win]
```

Windows workaround (see https://github.com/conda/conda-build/pull/4786):

```suggestion
  merge_build_host: True  # [win]
  missing_dso_whitelist:
    - '*/R.dll'           # [win]
```